### PR TITLE
Remove the full rebuild param. It is not actually used for anything.

### PR DESCRIPTION
**Pull request by cweagans**
The `$full_rebuild` param is present in many of the methods in va_gov_build_trigger, but despite being passed through several layers of abstraction in some places, it's not actually used for anything. It appears to have been added in c6511ac431b39f7b63e15715617aa16a23a4c362 when there was a distinction between a full rebuild and a partial rebuild, but that distinction no longer exists.

## Description

Relates to #_issueid_. (or closes?)

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As user _uid_ with _user_role_
1. Do this
   - [ ] Validate that
2. Then
   - [ ] Validate that
3. Then validate Acceptance Criteria from issue
   - [ ] a
   - [ ] b
   - [ ] c

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Sitewide program`
- [x] `Platform CMS Team`
- [ ] `Sitewide crew ` (leave Sitewide unchecked and check the specific team instead)
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS experience`
  - [ ] `⭐️ Public websites`
  - [ ] `⭐️ Facilities`
  - [ ] `⭐️ User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Review by ndouglas — APPROVED** [review]

**Review by ndouglas — APPROVED** [review]

File: docroot/modules/custom/va_gov_build_trigger/src/Plugin/VAGov/Environment/BRDGHA.php [hunk 1]
```diff
@@ -82,7 +82,8 @@ public static function create(ContainerInterface $container, array $configuratio
   /**
    * {@inheritDoc}
    */
-  public function triggerFrontendBuild(string $front_end_git_ref = NULL, bool $full_rebuild = FALSE) : void {
+
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_build_trigger/src/Plugin/VAGov/Environment/BRDGHA.php:85]: CS no gap between doc block and function.

**Review by swirtSJW — COMMENTED** [review]
Good call @cweagans 

Just one CS issue.

File: docroot/modules/custom/va_gov_build_trigger/src/Plugin/VAGov/Environment/BRDGHA.php (hunk 1 above)
Comment (cweagans) [reply docroot/modules/custom/va_gov_build_trigger/src/Plugin/VAGov/Environment/BRDGHA.php]: heh, whoops. this is what i get for using the ui to merge `main`.

**Review by swirtSJW — APPROVED** [review]
Thanks for clearing out the cruft.

**Pull request merged**